xrviz/__init__.py

```diff
@@ -1,3 +1,4 @@
+from .dashboard import Dashboard as XrViz
```

Comment:
Since only 1 line change is required, I have added it here.
Now we can do
```
import xrviz
xrviz.XrViz(data, ...).show()
```
as stated in https://github.com/intake/xrviz/issues/30. 
(replaced `XRViz` with `XrViz` to keep it consistent everywhere)